VirtoCommerce.Platform.Web/Extensions/HostingEnviromentExtension.cs

```diff
@@ -28,5 +28,21 @@ public static string MapPath(this IHostingEnvironment hostEnv, string path)
 
             return result;
         }
+
+        public static string MapContentPath(this IHostingEnvironment hostEnv, string path)
```

Comment:
This method is redundant you can use Path.GetFullPath("public.key"); and this code will search in content path by default